test/functional/message/message-formatter.js

```diff
@@ -175,6 +178,13 @@ QUnit.test( "should support ICU message format", function( assert ) {
 	assert.equal( like({ count: 3 }), "You and 2 others liked this" );
 });
 
+QUnit.test( "should support Bidi structured text", function( assert ) {
+	assert.equal(
+		Globalize( "he" ).messageFormatter( "helloArray", true )( "Beethoven", "Mozart" ),
+		"\u05e9\u05dc\u05d5\u05dd, \u200FBeethoven\u200F & \u200FMozart\u200F"
```

Comment:
Strange, on Windows if one saves some file as having either Unicode or utf-8 encoding using Notepad, it adds special BOM characters at the beginning, and I do not se any in 
message-formatter.js
But, after all, having sample of Hebrew or Arabic text isn't important that much, so I can just use English in this test file.